wpilibc/sim/src/RobotDrive.cpp

```diff
@@ -20,7 +20,8 @@ using namespace frc;
 
 const int RobotDrive::kMaxNumberOfMotors;
 
-static auto make_shared_nodelete(SpeedController* ptr) {
+static std::shared_ptr<SpeedController> make_shared_nodelete(
```

Comment:
Hm. It didn't do that locally. I guess `decltype(auto)` actually deduces the value type here, but `decltype(auto)` is still the correct thing to do in C++14.